src/Package/Impl/DataInspect/REnvironmentCollection.cs

```diff
@@ -6,6 +6,8 @@
 
 namespace Microsoft.VisualStudio.R.Package.DataInspect {
     internal class REnvironmentCollection : List<REnvironment>{
```

Comment:
What is the purpose of this class? If you just need to create a collection from JSON, you might as well just make a helper method.